transport.rb

```diff
@@ -0,0 +1,18 @@
+class Transport
+
+  CONVERT_TO_MINUTES = 60
+
+  attr_accessor :max_weight, :speed, :available
+
+  def initialize(max_weight, speed, available)
+    @max_weight = max_weight
+    @speed = speed
+    @available = available
+  end
+
+  def delivery_time(distance)
+    @time = distance / @speed 
```

Comment:
We use instance variables when we need it in several places inside of class or for some lazy loading functionality. Also, u already have defined getter for `speed`

Reply:
instance @time was deleted